lib/iterator.js

```diff
@@ -187,11 +193,12 @@ Iterator.prototype._prereturn = function (nodes) {
 
 Iterator.prototype._sortOrder = function (i) {
   var gt = this._gt || !this._start
-  return gt && this._start === i ? SORT_GT : SORT_GTE
+  return gt && this._start === i ? this._order.gt : this._order.gte
 }
 
-function byKey (a, b) {
+function byKey (a, b, reverse) {
   var k = b.key.localeCompare(a.key)
+  if (reverse) k = -1 * k
```

Comment:
I think `return (reverse ? -1 : 1) * (k || b.feed - a.feed)` would flip the order on conflict as well which i guess would be expected?

Reply:
Ah yeah it would make sense to reverse those conflicting ones as well. Fixed